README.md

```diff
@@ -6,6 +6,13 @@
 `usb-detection` allows you to listen for insert/remove events of USB devices on your system.
 
 
+## Deprecated
```

Comment:
I feel like this section is very easy to miss. Maybe some 🛑 emoji.

Any good examples to pull from?

Reply:
better?

I can't think of any good examples. Most things that I find are deprecated are marked as deprecated on npm and archived on github without much explanation